### ISSUE TITLE
Selenium Remote Server: java.net.MalformedURLException

### ISSUE THREAD

**Issue by Benuhx** [LearnLib/alex#58]
Version: Latest docker image avaible at 04.12.19 / 15:00 Uhr
Full log: [log041219.txt](https://github.com/LearnLib/alex/files/3922215/log041219.txt)
Docker-Compose:  [docker-compose.txt](https://github.com/LearnLib/alex/files/3922238/docker-compose.txt) (rename to .yml)

Test execution fails with Exception: java.net.MalformedURLException: no protocol: ./target/files/system/http:/selenium-hub:4444/wd/hub

Steps to reproduce:
1. Login to Alex
2. App-Settings, set Selenium Remote Server Url to http://selenium-hub:4444/wd/hub
3. Test Managment, open random test
4. Click Test Configure Button
5. Set this test run configuration:
![Screenshot (26)](https://user-images.githubusercontent.com/15267677/70149295-16240e80-16a8-11ea-8452-b24c5557913a.png)
6. Save
7. Execute test

Fails with
```
java.net.MalformedURLException: no protocol: ./target/files/system/http:/selenium-hub:4444/wd/hub
at java.base/java.net.URL.<init>(URL.java:657) ~[na:na]
at java.base/java.net.URL.<init>(URL.java:553) ~[na:na]
at java.base/java.net.URL.<init>(URL.java:500) ~[na:na]
at de.learnlib.alex.learning.entities.webdrivers.RemoteDriverConfig.createDriver(RemoteDriverConfig.java:80) ~[classes!/:na]
at de.learnlib.alex.learning.services.connectors.WebSiteConnector.get(WebSiteConnector.java:124) ~[classes!/:na]
at de.learnlib.alex.data.entities.actions.web.GotoAction.execute(GotoAction.java:67) ~[classes!/:na]
at de.learnlib.alex.data.entities.actions.web.WebSymbolAction.execute(WebSymbolAction.java:41) ~[classes!/:na]
at de.learnlib.alex.data.entities.SymbolAction.executeAction(SymbolAction.java:224) ~[classes!/:na]
at de.learnlib.alex.data.entities.SymbolActionStep.execute(SymbolActionStep.java:62) ~[classes!/:na]
at de.learnlib.alex.data.entities.Symbol.execute(Symbol.java:392) ~[classes!/:na]
at de.learnlib.alex.data.entities.ParameterizedSymbol.execute(ParameterizedSymbol.java:144) ~[classes!/:na]
at de.learnlib.alex.testing.entities.TestCaseStep.execute(TestCaseStep.java:102) ~[classes!/:na]
at de.learnlib.alex.testing.services.TestExecutor.executeStep(TestExecutor.java:233) ~[classes!/:na]
at de.learnlib.alex.testing.services.TestExecutor.executeTestCase(TestExecutor.java:167) ~[classes!/:na]
at de.learnlib.alex.testing.services.TestExecutor.executeTests(TestExecutor.java:70) ~[classes!/:na]
at de.learnlib.alex.testing.services.TestThread.run(TestThread.java:136) ~[classes!/:na]
```

**Comment by lachawoj** [comment]
I confirm the issue.

**Comment by abainczyk** [comment]
Sorry, I was setting environment variables wrong. Should be fixed with a5cfff969e7a27408b2dda22ff2a5f4456e967ae

**Comment by Benuhx** [comment]
Your fix works :)
